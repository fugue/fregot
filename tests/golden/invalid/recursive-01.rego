package recursive_01

foo {
    foo
}
