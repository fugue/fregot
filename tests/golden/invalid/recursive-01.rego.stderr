fregot (recursion check error):
  "recursive-01.rego" (line 3, column 1):
  rule is recursive:

    3| foo {
       ^^^

  This rule is recursive.
